google/cloud/bigquery/client.py

```diff
@@ -2875,11 +3045,22 @@ def list_rows(
             params["selectedFields"] = ",".join(field.name for field in selected_fields)
         if start_index is not None:
             params["startIndex"] = start_index
+        path = "%s/data" % (table.path,)
+        span_attributes = {
+            "path": path,
+            "page_size": page_size,
+            "page_token": page_token,
+            "start_index": start_index,
+        }
 
+        with create_span(
+            name="BigQuery.listRows", attributes=span_attributes, client=self
+        ):
+            api_request = functools.partial(self._call_api, retry, timeout=timeout)
```

Comment:
any particular reason for moving this into a variable instead of creating it inline?

Reply:
Has to be called in the return statement. However this does not actually call the API and I am fixing that issue in another PR:
https://github.com/googleapis/python-bigquery/pull/228